lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -251,6 +280,27 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    toggleEditMode(buttonType: ButtonType, group?: CardViewGroup) {
```

Comment:
you are hardcoding the notion of Tags, Categories, etc in the ADF that has no idea what is that... tags and categories are provided in the App... cc @MichalKinas